### PR TITLE
[scaling]: Added an OTLP layer for the runtime

### PR DESCRIPTION
**Pull request by iulianbarbu**
## Description of change

This PR was essentially a manual cherry-pick of #1041 functionality related to the OTLP layer added to the runtime.
This PR awaits the addition of the same logger service added by #1041.

## How has this been tested? (if applicable)

N/A, pending some more work.

### CODE REVIEW

File: deployer/src/args.rs [hunk 1]
```diff
@@ -19,6 +19,10 @@ pub struct Args {
     #[clap(long, default_value = "http://provisioner:3000")]
     pub provisioner_address: Endpoint,
 
+    /// Address to connect to the logger service
+    #[clap(long, default_value = "http://logger:8000")]
+    pub logger_uri: Endpoint,
```
Comment (chesedo) [inline deployer/src/args.rs:24]: Can we keep these in a separate PR to be able to merge this PR?

**Review by chesedo — COMMENTED** [review]
Looks good. We should make this PR mergable if possible so that we don't have to wait for the other blocks

File: deployer/src/args.rs (hunk 1 above)
Comment (chesedo) [reply deployer/src/args.rs:24]: Nvm, this was the last of our dependencies :man_facepalming:

**Comment by iulianbarbu** [comment]
Closing this in favor of #1139
